### ISSUE TITLE
prepend 'docker-' to repo name

### ISSUE THREAD

**Issue by lsm5** [projectatomic/rhel-push-plugin#6]
This will make it consistent with `docker-novolume-plugin` and `docker-lvm-plugin`.
Or we could remove `docker-` from the other two. Either way works.

**Comment by runcom** [comment]
I'm more in favor or removing docker- from the other but no big deal, @rhatdan wdyt?

**Comment by lsm5** [comment]
yup, NBD either way, and in rpms, i'm prepending `docker-` to subpackages if it doesn't exist. so `docker-lvm-plugin` and `docker-rhel-push-plugin` etc.

**Comment by runcom** [comment]
my point is that if we talk about them as _docker plugins_ then the `docker-` prefix is redundant so better take it off, but again, as you all wish .

I.E.

The `docker-lvm-plugin` can be just the `lvm-plugin`, same goes for the novolume one: `docker daemon --autorization-plugin=novolume` (also w/o the plugin suffix)

**Comment by rhatdan** [comment]
Remove them.  I would like to see if in the future we could make these auth plugins able to be used by more then docker.  It would be nice if we could take advantage of some via k8s running runc containers for example.  Or atomic push could use them to stop pushing rhel content.
